src/V3Ast.h

```diff
@@ -1384,6 +1384,9 @@ class AstNode {
     int m_cloneCnt;  // Mark of when userp was set
     static int s_cloneCntGbl;  // Count of which userp is set
 
+    // Region
+    int m_regionId;
+
```

Comment:
This should only be in the node type(s) that need it, not globally as would then have a large memory/performance impact.  E.g. in AstNodeAssign. Also shouldn't be an int (see below).